dmoj/executors/compiled_executor.py

```diff
@@ -63,6 +64,14 @@ def __call__(self, *args, **kwargs):
         return obj
 
 
+# Because BaseExecutor has ABCMeta as a metaclass, we create this class so
+# CompiledExecutor can have a proper metaclass. See
+# http://www.phyast.pitt.edu/~micheles/python/metatype.html for more
+# details.
+class _BaseExecutorCompiledExecutorMeta(abc.ABCMeta, _CompiledExecutorMeta):
```

Comment:
I think you don't need class. Just make `_CompiledExecutorMeta` inherit from `abc.ABCMeta`, which has to implicitly inherit from `type`. We don't have any other users of `_CompiledExecutorMeta`.